public/components/monitoring/model_deployment_table.tsx

```diff
@@ -98,8 +98,7 @@ export const ModelDeploymentTable = ({
             return (
               <EuiHealth className="ml-modelStatusCell" color="danger">
                 <div className="eui-textTruncate">
-                  <span style={{ fontWeight: 600 }}>Not responding</span> on {planningNodesCount} of{' '}
-                  {planningNodesCount} nodes
+                  <b>Not responding</b> on {planningNodesCount} of {planningNodesCount} nodes
```

Comment:
nit - I generally think we'd recommend `<strong>` over `<b>` for semantic purposes, but realistically t shouldn't matter.